tests/scripts/basic-build-test.sh

```diff
@@ -79,15 +79,15 @@ if [ ! -f "seedfile" ]; then
     dd if=/dev/urandom of="seedfile" bs=32 count=1
 fi
 
-# Step 2a - Unit Tests
+# Step 2a - Unit Tests (keep going even if some tests fail)
 perl scripts/run-test-suites.pl -v 2 |tee unit-test-$TEST_OUTPUT
 echo
 
-# Step 2b - System Tests
+# Step 2b - System Test  (keep going even if some tests fail)
```

Comment:
The s at the end of 'Test' has been removed. Typo?

Reply:
Yes. Fixed in #3181.